src/utils/projectService/getProjectById.go

```diff
@@ -28,11 +31,8 @@ func getById(ctx context.Context, sdkConfig sdkConfig.Config, projectId string)
 	}
 
 	project, err := projectResponse.Output()
-	if err != nil { // TODO parse meta data
-		if strings.Contains(err.Error(), "Invalid user input") {
```

Comment:
it does not return this error code now?

Reply:
It does, but after we started parsing meta data, it's useless to wrap the message here, because the validation is contained in the meta.